migrations/20220506135000_schema_breakdown_ddl.sql

```diff
@@ -0,0 +1,72 @@
+CREATE TABLE IF NOT EXISTS `type_def_types` (
+    `id` int UNSIGNED NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(255) NOT NULL,
+    `description` TEXT,
+    `type` ENUM('OBJECT', 'SCALAR', 'INTERFACE', 'ENUM', 'INPUT', 'UNION', 'DIRECTIVE') NOT NULL,
+    PRIMARY KEY (`id`),
+    UNIQUE KEY `name` (`name`)
```

Comment:
this probably is wrong, because schema is versioned, so you can have schema in service A being moved to service B with another type

Reply:
Yes, you are right c: ! Our goal is to navigate through the current schema (so yes, the latest). This PR changes are focused only on this.